login-fire-form.html

```diff
@@ -330,6 +330,49 @@
       'signedin': '_handleSignIn'
     },
 
+    observers: [
+      '_onAuthReady(_statusKnown)',
+    ],
+
+    _onAuthReady: function(ready) {
+
+      if (ready && navigator.credentials && navigator.credentials.preventSilentAccess && this.auto) {
+        var that = this;
+
+        // The new Credential Management API is available
+        if(that.debug) {
+          console.log('Credential Management API available');
+        }
+
+        // User already signedIn, no need to retrieve credentials
+        if (that.signedIn) {
+          return;
+        }
+
+        navigator.credentials.get({
+          password: true,
+          mediation: 'silent'
+        }).then(function (cred) {
+
+          if(that.debug) {
+            console.log('Cred found');
+            console.log(cred);
+          }
+
+          // Ensure they're not empty or user already signedIn while fetching credentials
+          if (!cred || that.signedIn) {
+            return Promise.resolve();
+          }
+
+          if (cred.type === 'password') {
+            that.signIn(cred.id, cred.password);
+          } 
```

Comment:
The auto attribute was set in the common-behavior because the credentials API could be also implemented with a federated authority (facebook, google, ...).

I don't know yet how to handle those federated authorities cause the firebase-auth element is set in the login-fire-button and the login-fire-form.

IMO, the login-fire root element or the common-behavior should call the credentials API to retrieve federated authorities or the id/password (This point need discussion)